sympy/physics/mechanics/kane.py

```diff
@@ -112,17 +160,65 @@ class KanesMethod(object):
     perform linearization and how to deal with dependent coordinates & speeds,
     and how do deal with bringing non-contributing forces into evidence.
 
+    References
+    ==========
+
+    .. [Kane1985] Kane, T., Levinson, D. Dynamics Theory and Applications. 1985
+       McGraw-Hill
+
     """
 
     def __init__(self, frame, q_ind, u_ind, kd_eqs=None, q_dependent=None,
-            configuration_constraints=None, u_dependent=None,
-            velocity_constraints=None, acceleration_constraints=None,
-            u_auxiliary=None):
+                 configuration_constraints=None, u_dependent=None,
+                 velocity_constraints=None, acceleration_constraints=None,
+                 u_auxiliary=None):
+        """Initializes a KanesMethod object.
```

Comment:
I like the idea of putting the parameters in the `__init__()` docstring but it appears this documentation is not picked up by Sphinx (unless I'm missing something). This can be verified by going to [Kane's online docstring documentation](http://docs.sympy.org/latest/modules/physics/mechanics/api/kane_lagrange.html?highlight=kanesmethod#sympy.physics.mechanics.kane.KanesMethod) and doing a quick search on "Please read the online documentation." which is `__init__`'s current docstring. Doing so will show that this docstring is nowhere to be found. I would suggest then that the parameters be moved back to the docstring for the class overall.